group-coordinator/src/main/java/org/apache/kafka/coordinator/group/streams/topics/InternalTopicManager.java

```diff
@@ -148,6 +148,19 @@ private static Map<String, Integer> decidePartitionCounts(final LogContext logCo
 
         decidedPartitionCountsForInternalTopics.putAll(repartitionTopics.setup());
 
+        enforceCopartitioning(topology, copartitionGroupsBySubtopology, log,
+            decidedPartitionCountsForInternalTopics, copartitionedTopicsEnforcer);
```

Comment:
nit:
```suggestion
        enforceCopartitioning(
            topology, 
            copartitionGroupsBySubtopology, 
            log,
            decidedPartitionCountsForInternalTopics, 
            copartitionedTopicsEnforcer
        );
```
documented here: https://kafka.apache.org/coding-guide
There are always exceptions like log-messages.

Reply:
Done